src/main/java/net/juniper/netconf/NetconfSession.java

```diff
@@ -229,10 +231,19 @@ public void loadXMLConfiguration(String configuration, String loadType) throws I
                 "</rpc>" +
                 NetconfConstants.DEVICE_PROMPT;
         lastRpcReply = getRpcReply(rpc);
+        updateLastRpcReplyObject();
```

Comment:
this is fine. Would it make sense to update both the string and object lastRpcReply(Object) in the same private method below?

Reply:
Yes, that makes a lot of sense. Will do that as soon as I’m in front of a PC.

Reply:
Will wait for this update before merging.

Reply:
OK, I've updated this. Two new private methods ...

`setLastRpcReply(String)` which updates the reply string and reply object, and

`setHelloReply(String)` which updates the reply string and hello object. It's only invoked the once, but added for consistency.